x/axelarnet/client/rest/tx.go

```diff
@@ -2,13 +2,14 @@ package rest
 
```

Comment:
do not change this file anymore (deprecated)

Reply:
done